app/views/layouts/application.html.erb

```diff
@@ -22,7 +22,7 @@
   <body class="govuk-template__body">
     <script>document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
 
-    <a href="#content" class="govuk-skip-link"><%= content_for?(:skip_link_message) ? yield(:skip_link_message) : "Skip to main content" %></a>
+    <a href="#main-content" class="govuk-skip-link"><%= content_for?(:skip_link_message) ? yield(:skip_link_message) : "Skip to main content" %></a>
```

Comment:
Is it worth using a [skip link component](https://components.publishing.service.gov.uk/component-guide/skip_link) here?